sunpycube/cube/datacube.py

```diff
@@ -554,20 +555,16 @@ class CubeSequence(object):
         one single Cube.
     """
 
-    def __init__(self, data_list, meta=None, common_axis=None):
-        if not all(isinstance(data, Cube) for data in data_list):
-            raise ValueError("data list should be of cube object")
+    def __init__(self, data_list, meta=None, common_axis=0):
         self.data = data_list
         self.meta = meta
         self.common_axis = common_axis
-        self.shape = tuple([len(data_list)] + list(data_list[0].shape))
-
-    @classmethod
-    def _new_instance(cls, data_list, meta=None):
-        """
-        Instantiate a new instance of this class using given data.
-        """
-        return cls(data_list, meta=meta)
+        try:
+            self.shape = tuple([len(data_list)] + list(data_list[0].shape))
+        except AttributeError as err:
+            warnings.warn("AttributeError " + str(err))
+        else:
+            self.shape = tuple([len(data_list)])
```

Comment:
Is the purpose of this try...except statement to make CubeSequence work for non-Cubes/arrays?